README.md

```diff
@@ -128,12 +128,9 @@ moment, so please share with us the resolution to any rough spots that you find.
 
 ## Usage
 
-### Running code
-
-Hydrogen adds a command "Hydrogen: Run" to the command palette when you're in any text editor. Press ⌘-⇧-P to open the command palette and type "hydrogen" — it'll come up.
-
-The "Hydrogen: Run" command is bound to the keyboard shortcut ⌘-⌥-↩ by default.
+Hydrogen adds commands for running code to the command palette when you're in any text editor. Press ⌘-⇧-P to open the command palette and type "hydrogen" and they will come up.
```

Comment:
Shorter sentences work better, e.g. *"Hydrogen provides a selection of commands for running code. Press [...]"*. There is no need to repeat what can be inferred from the context.